.vsts-ci.yml

```diff
@@ -78,6 +78,12 @@ extends:
         name: $(DncEngInternalBuildPool)
         image: 1es-windows-2022
         os: windows
+      policheck:
+        enabled: true
+      tsa:
+        enabled: true
+      binskim:
+        enabled: true
```

Comment:
Docs for the SDL tools: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/sdlanalysis/overview#behavior-in-pipelines-extending-official-template

### policheck
I believe this is correct to enable this tool for both official and unofficial builds. I don't see any problems with adding this here.

### tsa
When redoing the pipeline, we opted to not have TSA enabled. Why? Well, looking at the documentation:

> ⚠️: It depends on whether TSA is enabled or not. If TSA is enabled, bugs will be filed for the issues identified by the tool, and the build won't be failed for the issues. If TSA is not enabled, any issue identified by the tool will break the build, and the break setting cannot be overridden to false.

We opted to have the build fail for issues like this. TSA creates DevOps work items and our general workflow only reviews these once a week. It separates the changes from the actual SDL issues. If you merge a PR that fails the build, you know immediately and can react.

### binskim
This is not necessary to explicitly enable. It is enabled by default. It can be removed here.

Reply:
Just talked to Marc and TSA is now preferred. So, you can ignore my comment about it as the decision is outdated.